tests/test_supported_models.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+from templates.supported_models import MODEL_CONFIGS, ModelConfig, get_supported_models
+
+
+@pytest.fixture
+def sample_azure_deployment():
+    return {"type": "azure", "model_name": "gpt-35-turbo", "deployment_name": "azure"}
+
+
+@pytest.fixture
+def sample_openai_deployment():
+    return {"type": "openai", "model_name": "gpt-3.5-turbo", "deployment_name": ""}
```

Comment:
Do you think that we can remove the fixtures since they're each used once anyway? We could be defining these deployments directly in the specific test cases where they're needed (like you do for the last 2 tests)!